Toolset/palettes/script editor/behaviors/revsehandlerlistbehavior.livecodescript

```diff
@@ -145,7 +145,7 @@ command update
       # settings we sort alphabetically anyway, it probably doesn't make much difference to just sort here.
       # This will produce an ordering F, G, H, PF, PH, S, which may need to be adjusted using a custom search
       # function here.
-      sort tHandlers numeric by handlerTypeSortCallback(word 1 of each)
+      sort tHandlers by handlerTypeSortCallback(word 1 of each) & word 2 of each
```

Comment:
hmm... should we add a note somewhere that if we ever end up with more than 9 handler types we need to switch to A...Z instead of 1...N. Alternatively this could be split into two sort commands by sorting in handler name order first.